scripts/paternity-test.py

```diff
@@ -0,0 +1,173 @@
+import io
+import os
+import json # might not be needed
+from firecloud import fiss
+from firecloud.errors import FireCloudServerError
+import firecloud.api as fapi
+import numpy as np
+import pandas as pd
+import logging
+#import re <-- doesn't work on Terra
+
+# User-set variables
+BILLING_PROJECT_ID="biodata-catalyst"
+WORKSPACE="TSV-AFY"
+SUBDIRECTORY="\/case2\/"
```

Comment:
I don't yet understand the need for escapes. They can probably be avoided.

Reply:
Without escapes it will cause errors later down the line.

```
BILLING_PROJECT_ID="biodata-catalyst"
WORKSPACE="TSV-AFY -- Case 4 -- Top Cekrit"
SUBDIRECTORY="/case1/"
TABLE_NAME="CRAMs"
try:
    bucket = os.environ["WORKSPACE_BUCKET"]
    response = fapi.list_entity_types(BILLING_PROJECT_ID, WORKSPACE)
    if response.status_code != 200:
        print("Error in Firecloud, check your billing project ID and the name of your workspace.")
    else:
        print("Firecloud has found your workspace!")
        directory = bucket + SUBDIRECTORY
except NameError:
    print("Caught a NameError exception. This probably means you didn't restart the kernal after"
          " running the first block of code (the one with all the imports). Run it again, restart"
          " the kernal, then try running every block of code (including the import one) again.")
!gsutil ls $directory
```
Firecloud will report having found the workspace as it doesn't look for the subdirectory, but gstutil ls will throw a command error. \case1\ on the other hand will not even run as thinks we're trying to escape the end of the string.